src/transport/Transport.d.ts

```diff
@@ -21,7 +22,7 @@ export declare interface Transport {
 
     getBlockHeaderByHeight(height): Promise<BlockHeader>
 
-    getIdentityIdsByPublicKeyHash(publicKeyHashes: Buffer[]): Promise<Buffer[]>
+    getIdentityIdsByPublicKeyHash(publicKeyHashes: Buffer[]): Promise<GetIdentityIdsByPublicKeyHashesResponse>
```

Comment:
I would keep Buffers as a return value because the abstract transport shouldn't respond with objects of specific implementation.